Frameworks/UIKit/UIButton.mm

```diff
@@ -644,6 +612,15 @@ - (void)touchesCancelled:(NSSet*)touchSet withEvent:(UIEvent*)event {
 }
 
 /**
+ @Status Interoperable
+*/
+- (void)setFrame:(CGRect)frame {
+    [super setFrame:frame];
+    [_xamlButton setWidth:self.bounds.size.width];
+    [_xamlButton setHeight:self.bounds.size.height];
+}
+
+/**
  @Status Stub
```

Comment:
Is this correctly annotated  - should it be caveat or stubbed with UNIMPLEMENTED()